tests/client/json_api_client_test.py

```diff
@@ -12,32 +12,29 @@
 
 class JsonApiClientTest(unittest.TestCase):
     def test_pack_params(self):
-        self.assertDictEqual(JsonApiClient._pack_params({
+        self.assertItemsEqual(JsonApiClient._pack_params({
```

Comment:
Oh good point, I should not review code from my phone.
On Mon, Aug 1, 2016 at 16:23 Panupong (Ice) Pasupat <
notifications@github.com> wrote:

> In tests/client/json_api_client_test.py
> <https://github.com/codalab/codalab-cli/pull/551#discussion_r73071160>:
>
> > @@ -12,32 +12,29 @@
> >
> >  class JsonApiClientTest(unittest.TestCase):
> >      def test_pack_params(self):
> > -        self.assertDictEqual(JsonApiClient._pack_params({
> > +        self.assertItemsEqual(JsonApiClient._pack_params({
>
> `assertItemsEqual(a, b)` checks whether `sorted(a) == sorted(b)`, so the
> ordering is already discarded. On Aug 1, 2016 16:16, Stephen Koo <
> notifications@github.com> wrote:In tests/client/json_api_client_test.py:
> @@ -12,32 +12,29 @@ class JsonApiClientTest(unittest.TestCase): def
> test_pack_params(self): - self.assertDictEqual(JsonApiClient._pack_params({
> + self.assertItemsEqual(JsonApiClient._pack_params({
> Perhaps wrap both arguments to this assertion with set() since the order
> is just an implementation detail? —You are receiving this because you
> authored the thread.Reply to this email directly, view it on GitHub, or
> mute the thread.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/551/files/49ebb724208137fe42ff2a226b70f6a930e9b1c7#r73071160>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAlhtfkmPdpgbgYh4Sqg9iN0ZLSdbljkks5qbn_tgaJpZM4JaEsJ>
> .
>